<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZonePickDialog</class>
 <widget class="QDialog" name="ZonePickDialog">
  <property name="windowTitle">
   <string>Time zone</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="ExpandingListWidget" name="zoneList">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="searchLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="searchEdit">
       <property name="placeholderText">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="defaultButton"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ExpandingListWidget</class>
   <extends>QListWidget</extends>
   <header>ExpandingListWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
